iminuit/minimize.py

```diff
@@ -0,0 +1,109 @@
+from iminuit import Minuit
+import warnings
+
+
+class OptimizeResult(dict):
+    """Imitation of scipy.optimize.OptimizeResult"""
+    def __init__(self, **kwargs):
+        dict.__init__(self, **kwargs)
+    def __getattr__(self, name):
+        try:
+            return self[name]
+        except KeyError:
+            raise AttributeError(name)
+
+
+def minimize(fun, x0, args=(), method=None,
+             jac=None, hess=None, hessp=None,
+             bounds=None, constraints=(),
+             tol=None,
+             callback=None, options=None):
+    """Imitates the interface of the SciPy method of the same name in scipy.optimize.
+
+    For argument description, see scipy.optimize.minimize.
+
+    The only supported method is 'Migrad'.
+    """
+    from numpy import array
```

Comment:
Please put as `import numpy as np`.
IMO makes it clearer to read in the code below ... `array` is also a common variable name.

Reply:
I have to disagree for several reasons. Firstly, calling `array` repeatedly is faster than `np.array`, because the latter makes an extra map lookup. We are talking about implementation code in a short function, so it is not a problem that `array` is a common variable name.
As a compromise, I could this import `from numpy import array as np_array`.

Reply:
Proof
```
In [6]: timeit.timeit("a =np.array(0)", setup="import numpy as np")
Out[6]: 0.3645598888397217

In [7]: timeit.timeit("a =np_array(0)", setup="from numpy import array as np_arr
   ...: ay")
Out[7]: 0.3079559803009033
```